File: backend/server/src/main/java/com/groudina/ten/demo/services/CompetitonPinGeneratorImpl.java
package com.groudina.ten.demo.services;

import org.springframework.stereotype.Component;

@Component
public class CompetitonPinGeneratorImpl implements ICompetitionPinGenerator {
    @Override
    public String generate() {

        return String.valueOf(System.currentTimeMillis() - 1578000000000L);

    }
}

File: backend/server/src/main/java/com/groudina/ten/demo/services/CompetitonPinGeneratorImpl.kt
package com.groudina.ten.demo.services

import org.springframework.stereotype.Component

@Component
class CompetitonPinGeneratorImpl : ICompetitionPinGenerator {
    override fun generate(): String {
        return (System.currentTimeMillis() - 1578000000000L).toString()
    }
}